src/main/java/org/candlepin/subscriptions/capacity/CapacityReconciliationController.java

```diff
@@ -111,23 +114,25 @@ public void enqueueReconcileCapacityForOffering(String sku) {
   private void reconcileSubscriptionCapacities(Subscription subscription) {
     Offering offering = subscription.getOffering();
     var existingKeys = new HashSet<>(subscription.getSubscriptionMeasurements().keySet());
-    upsertMeasurement(subscription, offering.getCores(), "PHYSICAL", CORES)
+    upsertMeasurement(subscription, offering.getCores(), PHYSICAL, CORES)
         .ifPresent(existingKeys::remove);
-    upsertMeasurement(subscription, offering.getHypervisorCores(), "HYPERVISOR", CORES)
+    upsertMeasurement(subscription, offering.getHypervisorCores(), HYPERVISOR, CORES)
         .ifPresent(existingKeys::remove);
-    upsertMeasurement(subscription, offering.getSockets(), "PHYSICAL", SOCKETS)
+    upsertMeasurement(subscription, offering.getSockets(), PHYSICAL, SOCKETS)
         .ifPresent(existingKeys::remove);
-    upsertMeasurement(subscription, offering.getHypervisorSockets(), "HYPERVISOR", SOCKETS)
+    upsertMeasurement(subscription, offering.getHypervisorSockets(), HYPERVISOR, SOCKETS)
         .ifPresent(existingKeys::remove);
-    // existingKeys now contains only stale SubscriptionMeasurement keys (i.e. measurements no
-    // longer provided).
-    existingKeys.forEach(subscription.getSubscriptionMeasurements()::remove);
-    if (!existingKeys.isEmpty()) {
-      measurementsDeleted.increment(existingKeys.size());
-      log.info(
-          "Update for subscription ID {} removed {} incorrect capacity measurements.",
-          subscription.getSubscriptionId(),
-          existingKeys.size());
+    if (!offering.isMetered()) {
```

Comment:
I'm not sure if it makes sense to do this here or never use the reconcileSubscriptionCapacities method for the offering with metered instead.

Reply:
I'm not sure if this is required either but I think keeping it as is the safest route.